mithril-common/src/entities/signer.rs

```diff
@@ -68,6 +90,34 @@ impl SignerWithStake {
     }
 }
 
+impl PartialOrd for SignerWithStake {
```

Comment:
Same as above.

Reply:
This ensures that the `Certificate` hash is computed deterministically, which is not the case if the order is not guaranteed